test/e2e/marketplace_test.go

```diff
@@ -60,7 +60,7 @@ func TestMarketplace(t *testing.T) {
 	// run subtests
 	t.Run("marketplace-group", func(t *testing.T) {
```

Comment:
I don't see why we have this layer. With this implementation, test output will be similar to:
`TestMarketplace marketplace-group Cluster` vs `TestMarketplace marketplace-group csc-tests`. I don't see the value add of `marketplace-group`.